WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+import SwiftUI
+import AsyncImageKit
+import WordPressCore
+
+struct PluginListItemView: View {
+
+    private static let iconSize: CGFloat = 44
+
+    @ScaledMetric(relativeTo: .body) var descriptionFontSize: CGFloat = 14
+
+    private var iconURL: URL?
+    private var name: String
+    private var version: String
+    private var author: String
+    private var shortDescription: String
+    private var iconResolver: PluginIconResolver
+
+    init(plugin: InstalledPlugin, iconResolver: PluginIconResolver) {
+        self.iconURL = plugin.iconURL
+        self.name = plugin.name
+        self.version = plugin.version
+        self.author = plugin.author
+        self.shortDescription = plugin.shortDescription
+        self.iconResolver = iconResolver
+    }
+
+    var body: some View {
+        HStack(alignment: .top) {
+            CachedAsyncImage(urlResolver: iconResolver) { image in
+                image.resizable()
+            } placeholder: {
+                Image("site-menu-plugins")
+                    .resizable()
+            }
+            .frame(width: Self.iconSize, height: Self.iconSize)
+            .padding(.all, 4)
+
+            VStack(alignment: .leading, spacing: 0) {
+                Text(name)
+                    .lineLimit(1)
+                    .font(.headline)
+                    .foregroundStyle(.primary)
+
+                if !author.isEmpty {
+                    Text(Strings.author(author))
+                        .lineLimit(1)
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                }
+
+                Group {
+                    if shortDescription.isEmpty {
+                        Text(Strings.noDescriptionAvailable)
+                            .font(.system(size: descriptionFontSize).italic())
+                    } else if let html = renderedDescription() {
+                        Text(html)
+                    } else {
+                        Text(shortDescription)
+                            .font(.system(size: descriptionFontSize))
+                    }
+                }
+                .padding(.vertical, 4)
+
+                Text(Strings.version(version))
+                    .font(.caption)
+                    .foregroundStyle(.secondary)
+            }
+        }
+    }
+
+    func renderedDescription() -> AttributedString? {
+        guard var data = shortDescription.data(using: .utf8) else {
+            return nil
+        }
+
+        // We want to use the system font, instead of the default "Times New Roman" font in the rendered HTML.
+        // Using `.defaultAttributes: [.font: systemFont(...)]` in the `NSAttributedString` initialiser below doesn't
```

Comment:
I think it's OK to show a description, but I would limit it to 2 lines. It looks off when there are many lines with this tiny font. 

The first two lines might help someone identify what the plugin is about and if they want to "Read More", they'll tap to open the details. It's not unlike how the app renders excepts for posts.

<img width="320" alt="Screenshot 2025-01-30 at 5 18 31 PM" src="https://github.com/user-attachments/assets/cc0e98ee-0b30-45ad-8097-0c08e3aa43fe" />

Reply:
This part is likely to change as I'm developing the details screen. Do you mind using this design as it is for this PR, and I'll revisit it in follow up PRs?